packages/orchestrator-ui-components/src/rtk/api.ts

```diff
@@ -63,8 +63,11 @@ export const handlePromiseErrorWithCallback = <T>(
     });
 };
 
-export const handleGraphqlMetaErrors = (meta: WfoGraphqlErrorsMeta) => {
-    if (meta.errors && meta.errors.length > 0) {
+export const handleGraphqlMetaErrors = (
```

Comment:
I realize it is called in the SURF specific app:
`apps/wfo-ui-surf/rtk/endpoints/portsForChassis.ts` (no PR opened yet)

Because of that i am going to change some things here:
- add inline comment to explain the usage
- rename responseHasValidData to responseHasInvalidData and making it optional